### PR TITLE
Fix batchSend form validation on change Closes #531

### LINKED ISSUE #531: Batch Send Allows Form Submission With Invalid Amounts
The form shows error messages for amounts that are too low, but the send button is not deactivated, and the transaction is allowed to be submitted.

batch-send-amounts

### PR DESCRIPTION
**Pull request by garyghayrat**
fixes #531

### CODE REVIEW

**Review by apbendi — APPROVED** [review]
Working well for me. Nice work @garyghayrat

File: frontend/src/pages/AccountSend.vue [hunk 1]
```diff
@@ -741,6 +742,25 @@ function useSendForm() {
...
+      for (const batchSend of batchSends.value) {
+        if (validatedBatchSendForm) {
+          const { token, amount, receiver } = batchSend;
+          const isValidAmount = Boolean(amount) && isValidTokenAmount(amount, token);
+          const isValidToken = Boolean(token);
+          isValidRecipientPromises.push(isValidId(receiver));
+          if (isValidAmount !== true || !receiver || !isValidToken) validatedBatchSendForm = false;
+        }
+      }
+      await Promise.all(isValidRecipientPromises).then((results) => {
```
Comment (alexkeating) [inline frontend/src/pages/AccountSend.vue:757]: Small optimization, if `validatedBatchSendForm` is false we can skip this.

```
if (validatedBatchSendForm) {
  await Promise.all(...)
}

```

**Review by alexkeating — APPROVED** [review]
LGTM, one small optional optimization

File: frontend/src/pages/AccountSend.vue (hunk 1 above)
Comment (garyghayrat) [reply frontend/src/pages/AccountSend.vue]: Thanks! https://github.com/ScopeLift/umbra-protocol/pull/538/commits/fa463d673f6f8388afbc7847637c8d71718084b3

**Pull request merged**